### PR TITLE
Updated docs for cluster metadata map

### PR DESCRIPTION
**Pull request by jpapejr**
Folks will probably want to use more human-readable values in the dashboard for clusters (a customizable name vs a generated ID). The functionality is already there (because Razee is awesome) but the docs just don't mention it. This PR fixes that.

### CODE REVIEW

**Comment by alewitt2** [comment]
can you make this step 5 of this section. And label it optional. something like `5. Optional:`

**Comment by jpapejr** [comment]
Will do.

File: README.md [hunk 1]
```diff
@@ -318,6 +318,25 @@ With Watch Keeper set up in your cluster, you can retrieve deployment informatio
...
+    kind: ConfigMap
+    metadata:
+      labels:
+        razee/cluster-metadata: "true"
+        razee/watch-resource: debug
+      name: razee1-cluster-metadata
+      namespace: default
+    ````
+
+    After the ConfigMap is picked up (within a few moments), Razee will diplay this cluster on the dashboard using `razee-1` instead of the long ID value seen before. Note that it may take up to 2 minutes before the change can be seen in the dashboard.
```
Comment (alewitt2) [inline README.md]: typo: diplay

Comment (alewitt2) [reply README.md]: and it should only take up to 1 minute before change can be seen

**Comment by alewitt2** [comment]
thanks

**Pull request merged**